engines/base/src/main/java/org/alfresco/transform/base/registry/TransformRegistry.java

```diff
@@ -151,44 +142,77 @@ public void handleContextRefreshedEvent(final ContextRefreshedEvent event)
      * Load the registry on application startup. This allows Components in projects that extend the t-engine base
      * to use @PostConstruct to add to {@code transformConfigSources}, before the registry is loaded.
      */
-    @Async
-    @Retryable(include = {IllegalStateException.class},
-        maxAttemptsExpression = "#{${transform.engine.config.retry.attempts}}",
-        backoff = @Backoff(delayExpression = "#{${transform.engine.config.retry.timeout} * 1000}"))
     void initRegistryOnAppStartup(final ContextRefreshedEvent event)
     {
         customTransformerList = event.getApplicationContext().getBean(CustomTransformers.class).toList();
         retrieveConfig();
     }
 
     /**
-     * Recovery method in case all the retries fail. If not specified, the @Retryable method will cause the application
-     * to stop, which we don't want as the t-engine issue may have been sorted out in an hour when the next scheduled
-     * try is made.
+     * Takes the schedule from a spring-boot property
      */
-    @Recover
-    void recover(IllegalStateException e)
+    @Scheduled(cron = "${transform.engine.config.cron}")
+    public void retrieveEngineConfigs()
     {
-        logger.warn(e.getMessage());
+        logger.trace("Refresh TransformRegistry.");
+        retrieveConfig();
     }
 
     /**
-     * Takes the schedule from a spring-boot property
+     * Recovery mode method, executes only when there was failure during initial config retrieval process.
      */
-    @Scheduled(cron = "${transform.engine.config.cron}")
-    public void retrieveEngineConfigs()
+    @Scheduled(initialDelay = 1000, fixedDelayString = "#{${transform.engine.config.retry.timeout} * 1000}")
```

Comment:
1s might be not enough to give time for the initmethod. I suggest setting it to `#{${transform.engine.config.retry.timeout} * 1000}`